### DISCUSSION TITLE
-c: option requires an argument

### DISCUSSION THREAD

**Discussion by TracyMuso** [Homebrew/discussions#2144] [Getting started]
![D5710CB0-599A-4858-9A1C-A1EB10D2BBB2](https://user-images.githubusercontent.com/90801216/133673066-75e333b8-d6f0-4a00-9539-8ef7759c9063.jpeg)
![BD48D4BD-9E17-46C9-A93F-5501A710065F](https://user-images.githubusercontent.com/90801216/133673078-9be24460-4610-4a97-961d-6e361789485e.jpeg)
how can I clear this, is there an alternative method to installing homebrew? help please! I already tried the demoit thingy. it didn't work. Is there something I’m not doing right?

**Comment by SMillerDev** [comment]
When are you getting this? What did you try? What device are you using?

**Reply by TracyMuso** [reply]
It’s happening when I run /bin/bash -c...in Ubuntu 
I have a windows 10 Lenovo ideapad version 21H1 
OC 19043.1202

**Reply by SMillerDev** [reply]
So how are you running it in WSL?

**Reply by TracyMuso** [reply]
I pressed enter, then it said -c: option requires argument. So I failed to install it. I’ll attach a picture of it.
